### PR TITLE
[hotfix] - resolve the sql parse conflict

### PR DESCRIPTION
**Pull request by xiejiajun**
### What problem does this PR solve?
- 清理遗留的老版本SQL切分逻辑，完全使用最新的SqlSplitter进行SQL切分

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

How it Works:

### CODE REVIEW

**Review by wolfboys — COMMENTED** [review]
need further improvement.

File: streamx-flink/streamx-flink-shims/streamx-flink-shims-base/src/main/scala/com/streamxhub/streamx/flink/core/SqlCommandParser.scala [hunk 1]
```diff
@@ -35,31 +35,24 @@ object SqlCommandParser extends Logger {
     require(sql != null && sql.trim.nonEmpty, sqlEmptyError)
     val lines = SqlSplitter.splitSql(sql).filter(x => x != null && x.trim.nonEmpty)
```
[the remark this answers is not in the document]
Comment (xiejiajun) [reply streamx-flink/streamx-flink-shims/streamx-flink-shims-base/src/main/scala/com/streamxhub/streamx/flink/core/SqlCommandParser.scala]: Thank you for your suggestion, I have improved it, please review again.

**Comment by xiejiajun** [comment]
> need further improvement.

@wolfboys Thank you for your suggestion, I have improved it, please review again.

**Comment by wolfboys** [comment]
LGTM.

**Pull request merged**